docs/modules/components/pages/processors/ollama_chat.adoc

```diff
@@ -123,6 +133,170 @@ This field supports xref:configuration:interpolation.adoc#bloblang-queries[inter
 *Type*: `string`
 
 
+=== `response_format`
+
+The response format of generated type, the model must also be prompted to output the appropriate response type.
+
+
+*Type*: `string`
+
+*Default*: `"text"`
+
+Options:
+`text`
+, `json`
+.
+
+=== `max_tokens`
+
+The maximum number of tokens to predict and output.
+
+
+*Type*: `int`
+
+
+=== `temperature`
+
+The temperature of the model. Increasing the temperature will make the model answer more creatively.
+
+
+*Type*: `int`
+
+
+=== `num_keep`
+
+Specify the number of tokens from the initial prompt to retain when the model resets its internal context. By default, this value is set to 4. Use -1 to retain all tokens from the initial prompt.
```

Comment:
Under what circumstances does a reset occur? What does setting this value to `4` do?